root/layout/components/sidebar/GenreSidebar.js

```diff
@@ -9,14 +9,30 @@
 
 import React from 'react';
 
+import {withCatalystContext} from '../../../context';
+
 import LastUpdated from './LastUpdated';
+import RemoveLink from './RemoveLink';
+
+type Props = {|
+  +$c: CatalystContextT,
+  +genre: GenreT,
```

Comment:
FWIW, `genre: GenreT` was fine too, it is not necessary to accept subtypes with `+` for genre.

Reply:
We seem to use + pretty much everywhere, so I feel it'd be a bit weird to not use it here even if it's ok not to.

Reply:
`+` makes the field read-only, so you can't modify the props (because that's bad).

It also allows subtypes of `GenreT` for the property because of that, i.e. *because* the field isn't writable, but here it's only useful for making it read-only. https://flow.org/blog/2016/10/04/Property-Variance/

Reply:
Did not really mean it should changed, but anyway I got it wrong as I thought default was read-only but it is just type invariant.